src/libgraphviz/lib.rs

```diff
@@ -822,10 +833,12 @@ r#"digraph single_cyclic_node {
     #[test]
     fn hasse_diagram() {
         let labels = AllNodesLabelled(vec!("{x,y}", "{x}", "{y}", "{}"));
-        let r = test_input(LabelledGraph::new(
-            "hasse_diagram", labels,
-            vec!(edge(0, 1, ""), edge(0, 2, ""),
-                 edge(1, 3, ""), edge(2, 3, ""))));
+        let r = test_input(LabelledGraph::new("hasse_diagram",
+                                              labels,
+                                              vec!(edge(0, 1, ""),
+                                                   edge(0, 2, ""),
+                                                   edge(1, 3, ""),
+                                                   edge(2, 3, ""))));
```

Comment:
My 2 cents: I find it depends on the case, sometimes I prefer hanging indent, sometimes not, depending on various concerns. I think it'd be good to preserve what the author chose. I really *dislike* when all the code is bunched up against the right margin though, so I tend to prefer hanging idents if that's the only way to get "full" utilization.